### PR TITLE
Homework3: Fedor Polikarpov

### PR DESCRIPTION
**Pull request by FedorPolikarpov**

### CODE REVIEW

File: FzzBuzz.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Напишите программу, которая печатает цифры от 1 до 100,
+# но вместо чисел, кратных 3 пишет Fizz, вместо чисел кратных 5 пишет
+# Buzz, а вместо чисел одновременно кратных и 3 и 5 - FizzBuzz
+for i in range(1, 101):
+    if i % 3 == 0 and i % 5 == 0:
```
Comment (belovmd) [inline FzzBuzz.py]: 15?

**Review by belovmd — COMMENTED** [review]
1. Please don't use russian names for files

File: hello world [hunk 2]
```diff
@@ -0,0 +1 @@
+print("Hello world")
```
Comment (belovmd) [inline hello world]: what is it doing here? :)

File: кортежи.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+kort2 = (el for el in 'abc')
+lst2 = list(kort2)
+print(lst2)
+# Сделайте следующие присвоения одной строкой a = 'a', b=2, c=’python’.
+a, b, c = 'a', 2, 'python'
+
+# Создайте кортеж из одного элемента, чтобы при итерировании по этому
+# элементы последовательно выводились значения 1, 2, 3. Убедитесь что
+# len() исходного кортежа возвращает 1.
+lst3 = (123,)
```
Comment (belovmd) [inline кортежи.py]: it's not tuple with iterable first element

File: Упорядоченный список.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Дан список целых чисел. Требуется переместить все ненулевые
+# элементы в левую часть списка, не меняя их порядок,
+# а все нули - в правую часть.
+user_input = input('Введите элементы списка: ').split()
+zero = 0
+lenn = len(user_input)
+for i in range(lenn):
+    if int(user_input[i]) == 0:
+        zero += 1
+for i in range(zero):
```
Comment (belovmd) [inline Упорядоченный список.py]: you should use only one for loop according to task description

**Review by belovmd — COMMENTED** [review]
Please address these minor issues

File: hello world (hunk 2 above)
Comment (FedorPolikarpov) [reply hello world]: oops

File: FzzBuzz.py (hunk 1 above)
Comment (FedorPolikarpov) [reply FzzBuzz.py]: 15 multiple of 3 and 5. Output: FizzBuzz

File: Упорядоченный список.py (hunk 4 above)
Comment (FedorPolikarpov) [reply Упорядоченный список.py]: rdy

File: FzzBuzz.py (hunk 1 above)
Comment (FedorPolikarpov) [reply FzzBuzz.py]: oh, sry, I understood, rdy

File: кортежи.py (hunk 3 above)
Comment (FedorPolikarpov) [reply кортежи.py]: rdy

File: Lists [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+# следующий ['1a', '2a', '3a', '4a'].
+# Одной строкой удалите элемент  '2a'
+# из прошлого списка и напечатайте его.
+lst2 = [i + 'a' for i in '1234']
+print(lst2)
+print(lst2.pop(1))
+
+# Скопируйте список и добавьте в него элемент '2a'
+# так чтобы в исходном списке этого элемента не было.
+lst3 = copy.copy(lst2)
```
Comment (belovmd) [inline Lists:20]: [:] ?

**Review by belovmd — COMMENTED** [review]
9

**Pull request closed without merging**